tools/hlslpm/hlslpm.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import argparse
+import os
+from typing import List
+from issue import Issue, Issues
+import github
+
+
+def saveIssues(basePath, issues:List[Issue]):
+    for issue in issues:
+        filename = os.path.normpath(os.path.join(basePath, issue.issue_resourcePath.strip("/")))
+        os.makedirs(os.path.dirname(filename), exist_ok=True)
+
+        with open(filename, 'w') as f:
+            f.write(issue.body)
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(description="HLSL Project Manager")
+    parser.add_argument('--save', action='store_true', help="Saves before/after issues")
+    parser.add_argument('--update', action='store_true', help="Update github issues")
+
+    args = parser.parse_args()
+
+    if args.save and args.update:
+        print("Only one of --save or --update can be specified")
+    
+
+    print("Fetching data...")
+    gh = github.GH()
+    issues = Issues(gh)
+
+
+    tracked = list(issues.milestones.values()) + list(issues.workstreams.values())
```

Comment:
I see a lot of accesses to the raw dict inside of Issues. Might be good to hide that a bit with an accessor (something like `issues.get_milestone_list()`)_ that returns a `List[Issue]` since that seems to be the most common way to access it. Protects a bit in case we want to change the underlying data structure.

Reply:
These are lists now.  I'm trying to avoid too much abstraction and boilerplate code in this.